pkgs/dart_tooling_mcp_server/lib/src/mixins/analyzer.dart

```diff
@@ -268,7 +331,19 @@ base mixin DartAnalyzerSupport on ToolsSupport, LoggingSupport {
   static final analyzeFilesTool = Tool(
     name: 'analyze_files',
     description: 'Analyzes the entire project for errors.',
-    inputSchema: ObjectSchema(),
+    inputSchema: Schema.object(),
+  );
+
+  @visibleForTesting
+  static final resolveWorkspaceSymbolTool = Tool(
+    name: 'resolve_workspace_symbol',
```

Comment:
same question WRT the tool name. Should this be understandable by a human trying to decide whether to approve or deny the use of this tool?

Reply:
With the newest API version there is an additional field called "title" so I could add a user friendly title under there?

We should probably do this to all the tools

Reply:
Added tool annotations to give it a better name, PTAL